### ISSUE TITLE
Gráfico de entradas X despesas

### ISSUE THREAD

**Issue by rennerocha** [lhc/lhc-financas#5]
Usar o resultado de http://beta.lhc.rennerocha.com/status para gerar uma imagem com um gráfico de barras, mostrando o quanto já foi arrecadado e o quanto falta para arrecadar para atingirmos a meta do mês.

**Comment by Dimmy-Mess** [comment]
Olá rapaz! Vi seu GitHub no grupo do 0xE Hacker Clube, e acho que consegui fazer o que pediu! Era isso que esperava? Se sim, fala comigo que te passo o código

![balance_graph](https://user-images.githubusercontent.com/29147875/47475619-a5e36200-d7f2-11e8-95cb-e4d3f5aeea74.png)

**Comment by rennerocha** [comment]
Sim, era exatamente isso! Eu gostaria de colocar esse gráfico como uma imagem na wiki do LHC, então o ideal era que esse gráfico fosse gerado em um endpoint para que fosse possível fazer algo como:
`<img src="http://beta.lhc.rennerocha.com/grafico_status" />` para exibir a imagem.

Você pode fazer um PR com o código que você fez e assim vemos como adaptá-lo para fazer isso?
